setup.cfg

```diff
@@ -28,6 +28,7 @@ packages = find:
 setup-requires =
     setuptools >= 30.3.0
```

Comment:
I get the idea pbr should be added here (as well)

Reply:
I have the feeling that setup-requires should not be in setup.cfg anymore. The doc of pbr consistently puts it in setup.py, and having it at the two locations seems wrong. I'll try to remove it from setup.cfg and from setup.py when I'll be done cleaning the commit I am working on on a other branch.

Reply:
PEP-518; PEP-517.
Python packaging is a *mess* :(